static/.htaccess

```diff
@@ -10,6 +10,10 @@ Options -Indexes -MultiViews
 RewriteRule ^docs/8.3/(.*)$ /docs/$1 [R=302,L]
 RewriteRule ^optimize/3.11.0/(.*)$ /optimize/$1 [R=302,L]
 
+# Authorization to authentication
+RewriteRule ^optimize/next/apis-tools/optimize-api/optimize-api-authorization/(.*)$ /optimize/next/apis-tools/optimize-api/optimize-api-authentication/$1 [R=301,L]
+RewriteRule ^optimize/apis-tools/optimize-api/optimize-api-authorization/(.*)$ /optimize/apis-tools/optimize-api/optimize-api-authentication/$1 [R=301,L]
```

Comment:
@christinaausley I think this rewrite rule is causing the link-checking failure!

This file was moved in the `next` version (`docs/` folder), but it wasn't moved in the current version (`version-8.3/`). 

The server is redirecting this URL from the sitemap away from a page that does exist (current version/authorization) to a page that _doesn't_ exist (current version/authentication).

Reply:
☝️ I'm too slow 😆 I came here to say most changes are only applied to `/next/`. But I was too focused on debugging redirects. Thanks @pepopowitz!

Reply:
@akeller Are you fine with these changes going out with the minor release? If so, I'll just remove the redirect 👍

Reply:
Sure!